sdk/digitaltwins/azure-digitaltwins-core/MANIFEST.in

```diff
@@ -0,0 +1,6 @@
+include *.md
+include azure/digitaltwins/__init__.py
```

Comment:
We still needed to keep `include azure/__init__.py` :)
In otherwords - all layers of the namespace.

Reply:
Sure :) - Fixed